chingu-coders/Voyage2-Turtles-02
# small updates

## Purpose
* Match css styles
* Fix small bugs

## Approach/Fixes
* Newsfeed article toggles are visible now (changed font color)
* Added `rel="noopener"` to avoid tabnabbing
* Adjust page speed JS for smoother transitions

## Testing
* Check to make sure newsfeed toggle bars have visible text
* Check to make sure pagespeed works and when adding more details, the transitions are smooth
